tests/test_api.py

```diff
@@ -22,6 +22,14 @@ def test_direct_download_url_small_file(client):
     assert r.status_code == 200
     assert b"proximal colon" in r.data
 
+def test_direct_download_url_new_bucket_file(client):
+    #whil
```

Comment:
Remove.

Reply:
I understand that, I can revert those ones but I am hoping to keep them consistent.